Gulpfile.ts

```diff
@@ -137,13 +137,16 @@ const es2017LibrarySourceMap = es2017LibrarySource.map(function(source) {
 });
 
 const hostsLibrarySources = ["dom.generated.d.ts", "webworker.importscripts.d.ts", "scripthost.d.ts"];
+const es2015HostLibrarySources = ["dom.iterable.d.ts", "streams.d.ts",  "fetch.d.ts"]

```

Comment:
How should this be handled? `dom.iterable.d.ts` would still have to depend on iterables, which will depend on ES6 `Symbol`, which will depend on ES6 `Object`...